### PR TITLE
BC-7664 - rename room stores and room pages

### PR DESCRIPTION
**Pull request by wolfganggreschus**
# Short Description

The most necessary part of the old room namespace is renamed to give the new room concept the opportunity to be properly implied

## Links to Ticket and related Pull-Requests

BC-7664
## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team
- [x] DEV: Every new component is implemented having accessibility in mind (e.g. aria-label, role property)

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/router/routes.ts [hunk 1]
```diff
@@ -231,33 +231,47 @@ export const routes: Readonly<RouteRecordRaw[]> = [
...
 	},
-	{
-		path: `/rooms/:id(${REGEX_ID})/board`,
-		component: async () => (await import("@page-board")).ColumnBoardPage,
-		name: "rooms-board",
-		props: (route: RouteLocationNormalized) => ({
-			boardId: route.params.id,
-		}),
-	},
+	// TODO Is this redirect really necessary? How likely is it that there are bookmarks?
```
Comment (MartinSchuhmacher) [inline src/router/routes.ts:239]: imo we should get rid of that extra route

File: src/router/routes.ts [hunk 2]
```diff
@@ -231,33 +231,47 @@ export const routes: Readonly<RouteRecordRaw[]> = [
 		beforeEnter: createPermissionGuard(["news_edit"]),
 	},
 	{
-		path: `/rooms/:id(${REGEX_ID})`,
-		component: () => import("@/pages/rooms/RoomDetails.page.vue"),
-		name: "rooms-id",
-	},
-	{
+		// TODO BC-7822, BC-7823 target this route at new rooms page
```
Comment (MartinSchuhmacher) [inline src/router/routes.ts:234]: I would add to the comment "... and remove the redirect with the new component used" 
Just to be safe.

File: src/router/routes.ts (hunk 1 above)
Comment (uidp) [reply src/router/routes.ts:239]: Seems that there's still a reference in the legacy client:
https://github.com/hpi-schul-cloud/schulcloud-client/blob/90b9a5986160f04aabc510d2daa36bcfb5ceabc5/views/courses/edit-course.hbs#L50
That has to be updated then as well. Plus we have to remove the Ingress-entry from dof_app_deploy:
https://github.com/hpi-schul-cloud/dof_app_deploy/blob/8a3ad21db336c396ca6dec3c49821f09ce36b1cf/ansible/group_vars/all/x_ingress.yml#L90-L93

File: src/router/routes.ts (hunk 2 above)
Comment (uidp) [reply src/router/routes.ts]: Updated comment

File: src/router/routes.ts (hunk 1 above)
Comment (MartinSchuhmacher) [reply src/router/routes.ts:239]: then maybe we can add a comment here instead with something like "used in legacy and therefore necessary"?

**Review by uidp — DISMISSED** [review]

File: src/router/routes.ts (hunk 1 above)
Comment (uidp) [reply src/router/routes.ts:239]: Added the comment in line 234.

**Review by MartinSchuhmacher — APPROVED** [review]

File: src/router/routes.ts (hunk 1 above)
Comment (uidp) [reply src/router/routes.ts]: I created a ticket (BC-7877) and updated the comment.

**Pull request merged**